src/download-util.ts

```diff
@@ -19,14 +19,18 @@
  */
 import * as toolCache from '@actions/tool-cache';
 
+import { userAgentString } from './user-agent-util';
+
 /**
  * Downloads and extracts the tool at the specified URL.
  *
  * @url The URL of the tool to be downloaded.
  * @returns The path to the locally extracted tool.
  */
 export async function downloadAndExtractTool(url: string): Promise<string> {
-  const downloadPath = await toolCache.downloadTool(url);
+  const downloadPath = await toolCache.downloadTool(url, undefined, undefined, {
```

Comment:
Is this valid? I dont see this method signature in https://github.com/actions/toolkit/blob/master/packages/tool-cache/src/tool-cache.ts

Reply:
NVM, main master reference issue